# lidar parameters.
lidar_name: "HDL-64"
reverse_flag: true
h_min_ang: -180
h_max_ang: 180
num_hbeam: 1441

# line extraction parameters.
v_min_line_points: 5
v_seed_line_points: 2
v_least_thresh: 0.07
v_min_line_length: 0.15
v_predict_distance: 0.10
v_pts_missing_tolerance: 1
v_max_pts_gap: 1

h_min_line_points: 6
h_seed_line_points: 3
h_least_thresh: 0.06
h_min_line_length: 0.10
h_predict_distance: 0.12
h_pts_missing_tolerance: 1
h_max_pts_gap: 1

# line clustering parameters.
line_plane_distThreshold: 0.08

